src/main/java/com/hotels/beeju/core/ThriftHiveMetaStoreCore.java

```diff
@@ -98,6 +101,16 @@ public String getThriftConnectionUri() {
   public int getThriftPort() {
     return thriftPort;
   }
+  
+  /**
+   * @param thriftPort The Port to use for the Thrift Hive metastore, if not set then a port number will automatically be allocated.
+   */
+  public void setThriftPort(int thriftPort) {
+    if (thriftPort < 0) {
```

Comment:
Is it a good idea to allow for privileged ports (0-1023) to be assigned?

Reply:
I'll leave this decision with the end users of the library, don't think we need to force it one way or the other.